setup.js

```diff
@@ -44,15 +46,15 @@ function start_db(python_dir, on_success, on_error) {
     [resolve_local_file('scripts/start_database.py'), core.getInput('db_suffix')],
     { env: fb_env }
   );
-  return result.stderr.toString().length == 0 ? on_success(result.stdout.toString().trim('\n'), python_dir) : on_error(result.stderr.toString());
+  return result.status == 0 ? on_success(result.stdout.toString().trim('\n'), python_dir) : on_error(result.stderr.toString());
```

Comment:
Does this work correctly? I think it returns an object from `spawnSync`? https://nodejs.org/api/child_process.html#child_processspawnsynccommand-args-options

Reply:
I see status there
```
status [<number>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type) | [<null>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Null_type) The exit code of the subprocess, or null if the subprocess terminated due to a signal.
```

Reply:
Using stderr check doesn't work anymore since it's also triggered by warnings (trio warning currently)